Makefile

```diff
@@ -46,7 +46,8 @@ RELEASE_NOTES := $(TOOLS_DIR)/$(RELEASE_NOTES_BIN)
 # Binaries.
 KUSTOMIZE := $(TOOLS_BIN_DIR)/kustomize
 CONTROLLER_GEN := $(TOOLS_BIN_DIR)/controller-gen
-GOLANGCI_LINT := $(TOOLS_BIN_DIR)/golangci-lint
+# Need to use abspath so we can invoke this from subdirectories
+GOLANGCI_LINT := $(abspath $(TOOLS_BIN_DIR)/golangci-lint)
```

Comment:
Should we go ahead and add this for all of the tools to avoid potential for similar issues?

Reply:
Sure